### DISCUSSION TITLE
SearxNG Analytics w/ Umami?

### DISCUSSION THREAD

**Discussion by r0073d-l053r** [searxng/searxng#1745] [Q&A] [answered]
Is there a way to add a script into my html header of searxng that allows me to track analytics for how many people are using my engine? I'm currently using a Umami container for my analytics.
![text-1662266851531](https://user-images.githubusercontent.com/45528213/188299918-4fe8043a-ff19-474c-9484-40b3ffa7776b.png)

**Comment by return42** [comment, marked as answer]
The primary goal of SearXNG is to provide privacy to its users.  Any kind of analytics jeopardizes this goal and calls the whole project into question.  In this sense adding a script into html header is critical and not supported by us.

**Reply by r0073d-l053r** [reply]
Good to know. Thanks for the response. 🙏
